charts/cluster-scanner/tests/secret_test.yaml

```diff
@@ -60,3 +60,34 @@ tests:
     asserts:
       - isNull:
           path: data.cache_redis_password
+
+---
```

Comment:
I just tried locally and it seems these new tests are not actually running. I just tried to change input and expectations in order to see it fail and it doesn't 😕